### DISCUSSION TITLE
Noob question: where is config.yml?

### DISCUSSION THREAD

**Discussion by jphebert** [Lissy93/dashy#1149] [Help and Support]
HI!

My apologies for my new car smell, new to Docker & Dashy. I wish to edit the conf.yml file manually and can't find it. All I really want to do is to tweak the landing page and add widgets. May I ask for a copy/paste of what it should look like as well?

Am running Ubuntu server (latest) and installed Docker with the instructions from the Docker site. Where can I find the said file, or more precisely, where is the /public/conf.yml file located?  I find my docker stuff partly in "snap" folders if that helps.

I don't know the Linux folder system for what goes where, am a "classic Windows user" :)

Again, apologies for the basicness of the question!
Thanks!!!!

**Comment by lissy93** [comment]
Heya @jphebert 
In Docker everything is inside `/app` - so if you run `ls /app` you'll see files which look exactly the same as the code section of the repo. So try looking in `/app/public/conf.yml` :)

**Reply by jphebert** [reply]
Thanks for the quick response - I'm getting lost again - I'll put this on hold for another day :)

**Reply by sorousheta** [reply]
hi ..same problem
cant find path app docker!
just one i find in /var/lib/docker...... is that real path?

**Comment by Zsombe** [comment]
Hi!

I'm also a noob user, sorry. :) Bit i couldn't find my /app/public/conf.yml file. I'm using DeitPi. Could you help me please?

**Reply by coderunway** [reply]
Can you provide more info to help you better.  Some basic info to get started... 
1) Is your Dashy docker install successful? i.e. are you able to navigate to dashy home page ? 
2) What is the docker run command you gave to spin up your container ?

**Reply by jphebert** [reply]
I got it figured it out when I decided to take the easy route and run it on my Synology NAS and use this guide: https://mariushosting.com/synology-how-to-install-dashy-on-your-synology-nas/

**Reply by coderunway** [reply]
Great, yes Marius's blogs are a great source of info for Synology !  Glad it worked for you  Cheers

**Comment by ced4568** [comment]
I have the Dashy setup in docker and portainer correctly but I dont see the file path apps to be able to make changes to the conf.yml files. I trying to use the example homelab.

**Reply by CrazyWolf13** [reply]
Take a look here, dashy has some excellent written docs: https://dashy.to/docs/deployment

**Reply by React75** [reply]
I can't find it either. Reverted back to the old version.

**Reply by CrazyWolf13** [reply]
it"s located in /app/user-data
Inside the container and then mounted to the host system with a bind mount, this really depends how you set it.

If you mind sharing your docker command I may help.

**Reply by CrazyWolf13** [reply]
> I have the Dashy setup in docker and portainer correctly but I dont see the file path apps to be able to make changes to the conf.yml files. I trying to use the example homelab. 

Have you added a bind vomu with something similar like -v /some/path/on/the/host:/app/user-data

**Reply by MorganMarshall** [reply]
Where is "/app/user-data" on windows, I think full path will be easier to understand.

**Reply by MorganMarshall** [reply]
Update: my solution(windows 11) was to uses vscode's docker extension, then you can see the container files and go along with the regular instruction. Also found it in docker desktop files tab, right click and it allows you to edit in docker desktop.

**Reply by mgjsmith67** [reply]
Another noob in difficulty. I have Dashy installed via CasaOS and have searched high and low for the config file without success. HELP!

**Reply by mgjsmith67** [reply]
Another noob in difficulty. I have Dashy installed via CasaOS and have searched high and low for the config file without success. HELP!
